src/main/java/Command.java

```diff
@@ -0,0 +1,17 @@
+public abstract class Command {
+    /**
+     * Run the command on console.
+     * @param tasks Task list
+     * @param storage Handle file storage
+     * @param ui Show interaction
+     */
+
+    public abstract void run(TaskList tasks, Storage storage, UI ui);
+
+    /**
+     * Indicate end command.
+     * @return true if it is an end command
+     */
+
+    public abstract boolean isExit();
```

Comment:
Can consider giving this abstract method a default implementation that returns false, such that you only need to override the CommandBye class :)